isochrone/index.html

```diff
@@ -1,219 +1,72 @@
 <!DOCTYPE html>
```

Comment:
tbh i preferred the simplicity of a single file with all the stuff in it and minimal style crap (since im far from a front end dev) but you put in the effort so we'll go with it :smile: